data_summaries/Nectar.json

```diff
@@ -0,0 +1,65 @@
+{
+    "nectar": {
+        "Unique Dataset Identifier": "nectar",
+        "Dataset Name": "nectar",
+        "Paper Title": "Starling-7B: Improving LLM Helpfulness & Harmlessness with RLAIF",
+        "Dataset URL": "https://huggingface.co/datasets/berkeley-nest/Nectar",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/berkeley-nest/Nectar",
+        "Papers with Code URL": "",
+        "ArXiv URL": "",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "Nectar",
+        "Collection URL": "https://starling.cs.berkeley.edu/",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": ["RLHF"],
+        "Text Sources": [
+            "chat.lmsys.org",
+            "sharegpt.com",
+            "huggingface.co/datasets/Anthropic/hh-rlhf",
+            "huggingface.co/datasets/openbmb/UltraFeedback",
+            "huggingface.co/datasets/WizardLM/WizardLM_evol_instruct_V2_196k",
+            "huggingface.co/datasets/SirNeural/flan_v2"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-3.5-Turbo",
+            "OpenAI GPT-3.5-turbo-instruct",
+            "OpenAI GPT-4",
+            "LLama-2-7B-chat",
+            "Mistral-7B-Instruct"
+        ],
+        "Format": [
+            "Zero-shot",
+            "Chain-of-Thought"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "chat.lmsys.org",
+            "sharegpt.com",
+            "huggingface.co/datasets/Anthropic/hh-rlhf",
+            "huggingface.co/datasets/openbmb/UltraFeedback",
+            "huggingface.co/datasets/WizardLM/WizardLM_evol_instruct_V2_196k",
+            "huggingface.co/datasets/SirNeural/flan_v2"    
```

Comment:
Because `flan_V2` is in here and has some strict licenses, I would also put `Various` as a license (https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/constants/license_classes.json#L192) and cite in legal notes that it contains large mix of NC and Academic-Only licenses